src/wallet/wallet_rpc_server.cpp

```diff
@@ -335,32 +335,62 @@ namespace tools
   bool wallet_rpc_server::on_getbalance(const wallet_rpc::COMMAND_RPC_GET_BALANCE::request& req, wallet_rpc::COMMAND_RPC_GET_BALANCE::response& res, epee::json_rpc::error& er, const connection_context *ctx)
   {
     if (!m_wallet) return not_open(er);
+    if (req.all_accounts && !req.address_indices.empty())
```

Comment:
I wouldn't bother with this check. Just have all_accounts override.

Reply:
OK, updated